README.md

```diff
@@ -16,5 +16,30 @@ We will also be recording these presentations for those who cannot attend.
 - data: The dataset we will be using for presentations
 
 # Meeting Schedule 
+### [All Meetings Scheduled for 8:00 PM Eastern unless otherwise noted]
+
 - 2020-04-02: Introductions and Meeting Format
```

Comment:
Would you be able to  indicate that the chapter order will be as in the second edition of Advanced R, and provide the following URL for the second edition: https://adv-r.hadley.nz/  (though there is a link to the second edition from the first edition's website, the link is wrong https://github.com/hadley/adv-r/issues/1575)

Reply:
Cheers